src/StackDeploy/Stack.hs

```diff
@@ -73,16 +79,16 @@ perform = \case
       where
         doCreate :: Token -> m (AWS.Rs CF.CreateStack)
         doCreate token
-          = AWS.send
-          . configureStack operationFields instanceSpec token
-          . CF.createStack
-          $ toText name
+          = do
+            request <- prepareOperation operationFields instanceSpec token (CF.createStack (toText name))
```

Comment:
The code at https://github.com/mbj/stack-deploy/pull/50/files#diff-338cb999bc6dee49e4eabfa3e1349299R119 uses a `$` while this uses parenthesis.